.github/workflows/Sandpit_exs.yml

```diff
@@ -35,12 +35,10 @@ jobs:
 
       - name: Install dependency
         run: |
-          mamba install -c conda-forge -c openbiosim/label/main biosimspace python=3.8 ambertools gromacs "sire=2023.2.2" alchemlyb pytest pyarrow "typing-extensions!=4.6.0" openff-interchange pint=0.21 "rdkit<2023" "jaxlib>0.3.7" tqdm
+          mamba install -c conda-forge -c openbiosim/label/main biosimspace python=3.10 ambertools gromacs "sire=2023.3" "alchemlyb>=2.1" pytest openff-interchange pint=0.21 rdkit "jaxlib>0.3.7" tqdm
```

Comment:
Is this the right version for sire? Should it be `2023.3.2` if they want to pin to the last major release, or do they want `2023.4.0`?

Reply:
Good point. I'll check what's on their current branch.